darwin/client.py

```diff
@@ -1513,12 +1515,14 @@ def create_property(self, team_slug: Optional[str], params: Union[FullProperty,
             params=params,
         )
 
-    def update_property(self, team_slug: Optional[str], params: Union[FullProperty, JSONDict]) -> FullProperty:
+    def update_property(
+        self, team_slug: Optional[str], params: Union[FullProperty, JSONDict]
+    ) -> FullProperty:
         darwin_config = DarwinConfig.from_old(self.config)
         future_client = ClientCore(darwin_config)
 
         return update_property_future(
```

Comment:
Slight naming issue here, which is probably ok to leave, but a Future is a thing in itself, similar to a Javascript Promise, so worth bearing in mind in erm... _future_